portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IdentityEvents.testcase

```diff
@@ -267,6 +267,83 @@ definition {
 		}
 	}
 
+	@description = "Story: LRAC-10998 | Automation ID: LRAC-11380 | Test Summary: Check that the Identify event is not sent every time the user interacts with the site"
+	@priority = "3"
+	test CheckIdentityEventNotTriggeredAlwaysUserInteractsWithSite {
+		property proxy.server.enabled = "true";
+
+		task ("Add a new user") {
+			JSONUser.addUser(
+				userEmailAddress = "user1@liferay.com",
+				userFirstName = "user1",
+				userLastName = "user1",
+				userScreenName = "user1");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "user1@liferay.com");
+		}
+
+		task ("Sync the Contact Data") {
+			ACDXPSettings.syncNewContactData();
+		}
+
+		task ("Start Har recording") {
+			ProxyUtil.startHarRecording("identity");
+		}
+
+		task ("Sign in as new user") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "user1@liferay.com",
+				userLoginFullName = "user1 user1");
+		}
+
+		task ("Navigate to the connected site") {
+			ACUtils.navigateToSitePage(
+				pageName = "AC Page",
+				siteName = "Site Name");
+		}
+
+		task ("Get the time of the last update of the identity event") {
+			var localStorageItem1 = ACUtils.getLocalStorageItem(keyName = "ac_client_identity_last_updated_date");
+		}
+
+		task ("Visit the document") {
+			ACUtils.navigateToSitePage(
+				actionType = "View DM",
+				documentTitleList = "DM AC Title",
+				pageName = "AC Page",
+				siteName = "Site Name");
+		}
+
+		task ("Get the time of the last update of the identity event") {
+			var localStorageItem2 = ACUtils.getLocalStorageItem(keyName = "ac_client_identity_last_updated_date");
+		}
+
+		task ("Check that the time of the last update of the identity event has not changed") {
+			TestUtils.assertEquals(
+				actual = "${localStorageItem2}",
+				expected = "${localStorageItem1}");
+		}
+
+		task ("Navigate to the connected site") {
+			ACUtils.navigateToSitePage(
+				pageName = "AC Page",
+				siteName = "Site Name");
+		}
+
+		task ("Get the time of the last update of the identity event") {
+			var localStorageItem3 = ACUtils.getLocalStorageItem(keyName = "ac_client_identity_last_updated_date");
+		}
+
+		task ("Check that the time of the last update of the identity event has not changed") {
+			TestUtils.assertEquals(
+				actual = "${localStorageItem3}",
+				expected = "${localStorageItem1}");
+		}
+	}
```

Comment:
Hey Vini, I'm just curious. I saw that you compare the two identity values, first you visit the page it's at this moment the event of identity is triggered and you save the `localStorageItem1` and then you access the document and you save the `localStorageItem2`, and later you access the page again and save the `localStorageItem3` and compare the values again. Is this second comparison really necessary? Since the document and media are inside the page wouldn't make more sense instead of going back to the same page that the event was already triggered you visit a different page? What do you think?

Reply:
Hey @manuferreira, the bug happened for any navigation you did in DXP the `identity` event was sent, so it didn't matter if it was a page already accessed or another, the event was sent every time the page was loaded again, the same behavior as with page events. So in my test, I check if the `identity` event is being sent on two different page changes, first I access the DM and then I go back to the "home" page which is the AC Page and check that the event was not sent again in it too. Do you think it would be necessary to check on a third page as well?

Reply:
Hey @manuferreira, the bug happened for any navigation you did in DXP the `identity` event was sent, so it didn't matter if it was a page already accessed or another, the event was sent every time the page was loaded again, the same behavior as with page events. So in my test, I check if the `identity` event is being sent on two different page changes, first I access the DM and then I go back to the "home" page which is the AC Page and check that the event was not sent again in it too. Do you think it would be necessary to check on a third page as well?

Reply:
I like your approach @ViniLopes20 just to be sure, since we already had some bugs in this feature it seems.

Reply:
Ohh nice, I understand now thanks for the clarification Vini :clap: